src/xproc3/harness-lib.xpl

```diff
@@ -233,7 +233,7 @@
                   <p:with-input port="stylesheet" pipe="@formatter"/>
                   <p:with-option name="parameters" select="map{
                      xs:QName('force-focus'): $force-focus,
-                     xs:QName('report-theme'): $report-theme
+                     xs:QName('report-theme'): $html-report-theme
```

Comment:
`xs:QName('report-theme')` as the key remains unchanged because the XSLT parameter is still named `report-theme`